src/map/CMakeLists.txt

```diff
@@ -11,6 +11,9 @@ if( BUILD_SERVERS )
 message( STATUS "Creating target map-server" )
 file(GLOB MAP_HEADERS ${MAP_SOURCE_DIR}/*.hpp)
 file(GLOB MAP_SOURCES ${MAP_SOURCE_DIR}/*.cpp)
+# Combine testing of common and map
+file(GLOB TEST_COMMON_SOURCES ${CMAKE_SOURCE_DIR}/src/tests/common/*.cpp)
```

Comment:
Why is this need inside mapserver?

Reply:
Because if it would be in common the Common-Tests would run once for each server-type. Now they only run when map-server tests are executed

Reply:
Since we are testing "units", would it make sense to have its own executable and only run that?
As for now I do not really see the use in running all three servers in this CI step. You could simply remove 2 of the server starts and only leave one?

Reply:
For upcoming testing of char and login its required to execute them all in CI.
Each server-type runs it own tests. (eg. Login-Server will run login-server tests only)